### PR TITLE
List events improve

### PR DESCRIPTION
**Pull request by oVuVanChuyen**
This pullrequest will: Draw booked events improve
Screenshot:
[https://gyazo.com/b164082b8cda2a99a0674fa625b74133](https://gyazo.com/b164082b8cda2a99a0674fa625b74133)

[https://gyazo.com/690c1b748fdb75b0b256f07542f27da2](https://gyazo.com/690c1b748fdb75b0b256f07542f27da2)

[https://gyazo.com/5af3c13f976ccc38fdb4127ce310c387](https://gyazo.com/5af3c13f976ccc38fdb4127ce310c387)

### CODE REVIEW

File: app/scripts/app.js [hunk 1]
```diff
@@ -75,6 +75,7 @@
       .when('/events', {
         templateUrl: 'views/events/index.html',
         controller: 'EventsController',
+        controllerAs: 'vm'
```
Comment (yeuem1vannam) [inline app/scripts/app.js]: Use meaningful name please.

File: app/scripts/services/calendar_service.js [hunk 2]
```diff
@@ -0,0 +1,344 @@
+/*
+*  AngularJs Fullcalendar Wrapper for the JQuery FullCalendar
```
Comment (xuanduc987) [inline app/scripts/services/calendar_service.js]: bower plz
https://github.com/angular-ui/ui-calendar

**Comment by xuanduc987** [comment]
:ok:

**Comment by yeuem1vannam** [comment]
:o:K

**Pull request merged**